Pod/Classes/UI/CardTextField.swift

```diff
@@ -217,9 +217,9 @@ public class CardTextField: UITextField, NumberInputTextFieldDelegate {
         monthTextField?.deleteBackwardCallback = {_ -> Void in self.numberInputTextField?.becomeFirstResponder()}
         yearTextField?.deleteBackwardCallback = {_ -> Void in self.monthTextField?.becomeFirstResponder()}
         
-        let textFields: [UITextField?] = [numberInputTextField,cvcTextField,monthTextField,yearTextField]
+        let textFields: [UITextField?] = [numberInputTextField, cvcTextField, monthTextField, yearTextField]
         textFields.forEach({
-            $0?.keyboardType = .NumberPad
+            $0?.keyboardType = keyboardType
```

Comment:
I'd probably not make this customizable - developers will have to select NumberPad in interface builder whenever they integrate one and I don't see a chance that anything but a NumberPad keyboard will be required here.